### PR TITLE
Add custom labels support

### PR DESCRIPTION
**Pull request by Baldinof**
Introduce a new `label name value` directive ta add extra labels on all metrics.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,11 @@ These are the (optional) parameters that can be used:
   - **address** - the address where the metrics are exposed, the default is `localhost:9180`
   - **path** - the path to serve collected metrics from, the default is `/metrics`
   - **hostname** - the `host` parameter that can be found in the exported metrics, this defaults to the label specified for the server block
+  - **label** - Custom label to add on all measures.
```
Comment (miekg) [inline README.md]: measure -> metrics

**Comment by miekg** [comment]
also travis failures

**Comment by Baldinof** [comment]
I'll squash the fixup commit if it's ready for you.

File: setup.go [hunk 2]
```diff
@@ -158,6 +172,18 @@ func parse(c *caddy.Controller) (*Metrics, error) {
 					return nil, c.Err("prometheus: address and use_caddy_addr options may not be used together")
 				}
 				metrics.useCaddyAddr = true
+			case "label":
+				args = c.RemainingArgs()
+				if len(args) != 2 {
+					return nil, c.ArgErr()
+				}
+
+				fmt.Print(args)
```
Comment (miekg) [inline setup.go]: fmt can go

Comment (Baldinof) [reply setup.go]: 😱 indeed!

**Comment by Baldinof** [comment]
Any news here? @miekg

**Comment by Baldinof** [comment]
Thank you :)

**Pull request merged**